File: java/q4.java
import java.util.*;

public class Q4 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int maxSeq = 0, atual = 0;
        for (int i = 0; i < N; i++) {
            int temp = sc.nextInt();
            if (temp > 0) {
                atual++;
                maxSeq = Math.max(maxSeq, atual);
            } else {
                atual = 0;
            }
        }
        System.out.println(maxSeq);
    }
}

File: python/q4.py
while True:
    line = input().strip()
    if line != "":
        N = int(line)
        break

while True:
    line = input().strip()
    if line != "":
        temps = list(map(int, line.split()))
        break

max_seq = atual = 0
for t in temps:
    if t > 0:
        atual += 1
        max_seq = max(max_seq, atual)
    else:
        atual = 0

print(max_seq)
